apps/admin-gui/src/app/shared/components/dialogs/add-user-ext-source-dialog/add-user-ext-source-dialog.component.html

```diff
@@ -2,18 +2,21 @@ <h1 mat-dialog-title>{{'DIALOGS.ADD_USER_EXT_SOURCE.TITLE'|translate}}</h1>
 <div class="dialog-container user-theme" mat-dialog-content>
   <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
   <div *ngIf="!loading">
-    <mat-form-field appearance="standard" class="center-self w-100">
+    <mat-form-field class="center-self w-100">
       <input
         [formControl]="loginControl"
         matInput
-        placeholder="{{'DIALOGS.ADD_USER_EXT_SOURCE.LOGIN'|translate}}"
+        placeholder="{{'DIALOGS.ADD_USER_EXT_SOURCE.LOGIN' | translate}}"
         required>
+      <mat-error>
+        {{'DIALOGS.ADD_USER_EXT_SOURCE.EMPTY_LOGIN' | translate}}
+      </mat-error>
```

Comment:
In this dialog you kind messed up with that you didnt followed up the convention as in other dialogs. 
1. no ":" in the placeholder please
2. external source- when you not choose anything then it change color to red, but no mat error appears
3. in other dialog you use markAllAsTouched method in onInit, please follow that convention when you used in dialogs before this. Because now when you open this dialog you cannot click to add and nothing is red so from the first look it is looking like its bugged